### DISCUSSION TITLE
Are the "each" and "if" helpers missing from the docs?

### DISCUSSION THREAD

**Discussion by tuan-lm97** [mockoon/mockoon#1508] [Help] [answered]
The "each" and "if" helpers is mentioned in [the docs](https://mockoon.com/docs/latest/templating/mockoon-helpers) all the time.
But they lack detail guide like others helpers.
I was about to create an issue, but just want to do some sanity check first.
Am I missing it some where?

**Comment by 255kb** [comment, marked as answer]
Mockoon is using Handlebars which offers these [helpers](https://handlebarsjs.com/guide/builtin-helpers.html) (among others) out of the box: `each`, `if`, `lookup`, `with`. 
We didn't document this for obvious reasons, but I can make it clearer in the docs if necessary.

**Reply by tuan-lm97** [reply]
Oh yeah, now I see it in the docs "Besides Handlebars built-in helpers ...".
This can be a bit easy to miss it.
Maybe we can make it clearer by put the above link to the docs

**Reply by 255kb** [reply]
sure, I will clarify this on the helpers page.
